app/helpers/admin_api_helper.rb

```diff
@@ -11,6 +11,10 @@ def authenticate
     end
   end
 
+  def already_created_error(_exception)
+    render json: { error: "Record not unique" }, status: :conflict
```

Comment:
Very minor comment:

If you've using "Record not unique", might be nice to amend line 23 so that's it's "Record not found".

Reply:
Thanks, I've done that